opre_ops/ops_site/migrations/0005_person.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.2.6 on 2021-09-10 17:02
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ops_site', '0004_role'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Person',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('first_name', models.CharField(max_length=100)),
+                ('last_name', models.CharField(max_length=100)),
```

Comment:
As a team, we know there are important diversity and inclusion reasons to not have any minimum character lengths for people's names. Names come in lots of different shapes and sizes and one- or two- character names are common in a lot of cultures! 

I did some research to see if there are any similar considerations we should think about when it comes to `max_length`. 

* The [UK Government Data Standards Catalogue](https://webarchive.nationalarchives.gov.uk/ukgwa/+/http://www.cabinetoffice.gov.uk/media/254290/GDS%20Catalogue%20Vol%202.pdf) suggests a maximum of 35 characters each for forename/surname or 70 characters for combined names, and we are well above that number. 
* You can find articles on the internet about people with exceptionally long names, but those are "Guinness World Records" style exceptions and don't seem relevant to our userbase of fewer than ~100 people. 

I couldn't find anything to suggest any diversity/inclusion concerns with setting this proposed `max_length`.